indi-atik/atik_ccd.cpp

```diff
@@ -56,34 +58,47 @@ void ATIK_CCD_ISInit()
         iAvailableCamerasCount = 0;
         std::vector<std::string> cameraNames;
 
-        iAvailableCamerasCount = ArtemisDeviceCount();
-        if (iAvailableCamerasCount > MAX_DEVICES)
-            iAvailableCamerasCount = MAX_DEVICES;
+        IDLog("Atik Cameras API V%d DLL V%d initializing.", ArtemisAPIVersion(), ArtemisDLLVersion());
```

Comment:
The diff is weird here, but I just added a loop retrying the device detection.
The loop instantiating cameras doesn't count if the enumerated count is zero.